src/uiSelectMatchDirective.js

```diff
@@ -10,7 +10,10 @@ uis.directive('uiSelectMatch', ['uiSelectConfig', function(uiSelectConfig) {
 
       // Gets theme attribute from parent (ui-select)
       var theme = tElement.parent().attr('theme') || uiSelectConfig.theme;
-      var multi = tElement.parent().attr('multiple');
+
+      var multi = angular.isDefined(tElement.parent().attr('multiple')) ||
```

Comment:
Probably worth caching `tElement.parent()` above and rewriting to remove duplicate calls to `.parent()`